travvy/src/components/Home.vue

```diff
@@ -79,12 +79,12 @@
     <!-- data binding destination suggestions corresponding data object below -->
     <h2>Find your next travel destination</h2>
 
-    <button v-on:click="reverseMessage" class="nextDestination">Toronto<br><br><img src="../images/toronto.jpg" alt="Picture of Toronto" class ="cityimage"/></button>
-    <button v-on:click="reverseMessage" class="nextDestination">New York City<br><br><img src="../images/newyorkcity.jpg" alt="Picture of New York City" class ="cityimage"/></button>
-    <button v-on:click="reverseMessage" class="nextDestination">Rome<br><br><img src="../images/rome.jpg" alt="Picture of Rome" class ="cityimage"/></button>
-    <button v-on:click="reverseMessage" class="nextDestination">Rio De Janeiro<br><br><img src="../images/rio.jpg" alt="Picture of Rio de Janeiro" class ="cityimage"/></button>
-    <button v-on:click="reverseMessage" class="nextDestination">Paris<br><br><img src="../images/paris.jpg" alt="Picture of Paris" class ="cityimage"/></button>
-    <button v-on:click="reverseMessage" class="nextDestination">London<br><br><img src="../images/london.jpg" alt="Picture of London" class ="cityimage" /></button>
+    <button v-on:click="nextDest('Toronto')" class="nextDestination">Toronto<br><br><img src="../images/toronto2.jpg" alt="Picture of Toronto" class ="cityimage"/></button>
+    <button v-on:click="nextDest('New York City')" class="nextDestination">New York City<br><br><img src="../images/newyorkcity2.jpg" alt="Picture of New York City" class ="cityimage"/></button>
+    <button v-on:click="nextDest('Rome')" class="nextDestination">Rome<br><br><img src="../images/rome2.jpg" alt="Picture of Rome" class ="cityimage"/></button>
+    <button v-on:click="nextDest('Rio de Janeiro')" class="nextDestination">Rio De Janeiro<br><br><img src="../images/rio2.jpg" alt="Picture of Rio de Janeiro" class ="cityimage"/></button>
+    <button v-on:click="nextDest('Paris')" class="nextDestination">Paris<br><br><img src="../images/paris2.jpg" alt="Picture of Paris" class ="cityimage"/></button>
+    <button v-on:click="nextDest('London')" class="nextDestination">London<br><br><img src="../images/london2.jpeg" alt="Picture of London" class ="cityimage" /></button>
```

Comment:
I'm glad all these images will be the same size and enhance the appearance of these buttons from sprint 3!

Reply:
sweet! are they new pictures or just reformatted of the original?

Reply:
is this to populate the dropdown? looks great

Reply:
They are new pictures and yes it populates the dropdown!